CONTRIBUTING.md

```diff
@@ -199,5 +199,44 @@ $ yarn start --disable-host-check --public=$(echo ${server.dev-server} | sed -e
 ```
 
 ### Che server a.k.a WS master
+There is a [devfile](https://github.com/eclipse/che/blob/master/devfile.yaml) for development of Che server in Che.
```

Comment:
Could you change a hyperlink to the devfile on `[devfile](devfile.yaml)`?

Reply:
I think we should leave at least one full link to the repository.